src/map/utils/mobutils.cpp

```diff
@@ -189,6 +189,221 @@ namespace mobutils
+                else if (level <= 40)
+                    sJobStat = std::floor(stat / (3.1f - 0.075f * (level - 32)));
+                else if (level <= 45)
+                    sJobStat = std::floor(stat / (2.5f - 0.025f * (level - 40)));
+                else if (level <= 49)
+                    sJobStat = std::floor(stat / (2.35f - 0.04f * (level - 44)));
+                else
+                    sJobStat = std::floor(stat / 2);
+                break;
+
+            case 3: // C
+                if (level <= 30)
+                    sJobStat = std::max((std::floor(stat / (4.5f - 0.15f * (level - 26)))), 2.0f);
+                else if (level <= 40)
+                    sJobStat = std::floor(stat / (3.28f - 0.001f * (level - 30)));
+                else if (level <= 45)
+                    sJobStat = std::floor(stat / (2.6f - 0.025f * (level - 40)));
+                else if (level <= 49)
+                    sJobStat = std::floor(stat / (2.1f - 0.2f * (level - 49)));
+                else
+                    sJobStat = std::floor(stat / 2);
+                break;
+
+            case 4: // D
+                if (level <= 30)
+                    sJobStat = std::max((std::floor(stat / (5.0f - 0.05f * (level - 21)))), 1.0f);
+                else if (level <= 40)
+                    sJobStat = std::floor(stat / (3.2f - 0.001f * (level - 29)));
+                else if (level <= 45)
+                    sJobStat = std::floor(stat / (3.5f - 0.08f * (level - 32)));
+                else if (level <= 49)
+                    sJobStat = std::floor(stat / (3.25f - 0.045f * (level - 32)));
+                else
+                    sJobStat = std::floor(stat / 2);
+                break;
+
+            case 5: // E
+                if (level <= 30)
+                    sJobStat = std::max((std::floor(stat / (3.8f - 0.1f * (level - 32)))), 1.0f);
+                else if (level <= 40)
+                    sJobStat = std::floor(stat / (3.8f - 0.15f * (level - 32)));
+                else if (level <= 45)
+                    sJobStat = std::floor(stat / (2.7f - 0.075f * (level - 40)));
+                else if (level <= 49)
+                    sJobStat = std::floor(stat / (2.7f - 0.05f * (level - 45)));
+                else
+                    sJobStat = std::floor(stat / 2);
+                break;
+
+            case 6: // F
+                if (level <= 30)
+                    sJobStat = std::max((std::floor(stat / (4.0f - 0.15f * (level - 35)))), 1.0f);
+                else if (level <= 40)
+                    sJobStat = std::floor(stat / (4.0f - 0.15f * (level - 30)));
+                else if (level <= 46)
+                    sJobStat = std::floor(stat / (3.0f - 0.1125f * (level - 40)));
+                else if (level <= 49)
+                    sJobStat = std::floor(stat / (3.0f - 0.07f * (level - 40)));
+                else
+                    sJobStat = std::floor(stat / 2);
+                break;
+
+            case 7: // G
```

Comment:
I feel like we ought to say something in a comment somewhere about what scale is being used, considering elsewhere in the core pxi/dsp/tpz/lsb all used an `A+` through `E` scale instead of an `A` through `G` scale and we wouldn't want anyone in the future mixing the 2 up. I'm unsure what to say exactly though. Maybe we should link back to some of Jimmayus's research for it just above the switch - anyone curious can just go look then. Opinions/ideas welcome.

Reply:
I can link the studiogobli rank chart in the code if we want to show the A-G scale. All JP websites use this scale, I didn’t think of this until you mentioned it.

Reply:
While I have you in this code block should I remove all the last else statements on each case and just put it during the check for zone since anything over 50 is the same formula. It would stop this from running on mobs over level 50 and just default to stat/2.

Reply:
Fixed